### DISCUSSION TITLE
Unable to Specify a Different Schema in PostgreSQL Connection Using YAML Configuration

### DISCUSSION THREAD

**Discussion by sinjup** [Dataherald/dataherald#155] [Q&A]
We could successfully establish a connection to the PostgreSQL database using the following YAML update:
environment:
  DB_HOST: "localhost"     # Replace with your PostgreSQL server hostname or IP
  DB_PORT: "5432"          # PostgreSQL default port
  DB_NAME: "your_database" # Your PostgreSQL database name
  DB_USER: "postgres_uname"      # Your PostgreSQL database username
  DB_PASSWORD: "postgres_pword"
  
However, by default, the connection is made to the 'public' schema. We attempted to specify a different schema using DB_SCHEMA: "my_schema" in the YAML configuration, 
but it did not produce the desired outcome. Please advice.

**Comment by jcjc712** [comment]
Hello, when you use the endpoint in v0.0.2 POST `/api/v1/database-connections` or in v0.0.1 POST `/api/v1/databases` you can specify the schema in the `connection_uri` field like this `?options=-csearch_path=<schema-name>`
For example:
`"connection_uri": postgresql+psycopg2://<user>:<password>@<host>:5432/my-database?options=-csearch_path=my-schema`

I hope this helps
